CHANGELOG

```diff
@@ -1,3 +1,72 @@
+== OpenDataPlane (1.19.0.0)
+=== Summary of Changes
+ODP v1.19.0.0 is the official Tiger Moth final release. It incorporates final
+implementation changes and bug fixes and completes the Tiger Moth ODP
+development cycle.
+
+==== APIs
+No changes for this release. The Tiger Moth API was frozen in ODP v1.18.0.0.
+
+==== C++ Test Improvements
+The $$C++$$ test included in the ODP validation suite now uses `cout` instead
```

Comment:
Do we need the '$$' in '$$C++$$' or... ?

Reply:
@bogdanPricope @Bill-Fischofer-Linaro https://github.com/asciidoctor/asciidoctor/issues/1208

Reply:
Thanks for that pointer. I'll use it, though {cpp} in the text still looks a bit strange"